### DISCUSSION TITLE
GET calls to 'http://169.254.169.254/metadata/instance/compute' appear after using this library

### DISCUSSION THREAD

**Discussion by nabeel-servcorp** [datalust/seq-tickets#2152] [Help]
We have started seeing HTTP GET calls to http://169.254.169.254/metadata/instance/compute in our Seq logs, after we introduced this library for Seq tracing. Why is it so and can it be prevented?

On a side note, all these logs are logged as Error, with no other information available.

**Comment by liammclennan** [comment]
Hi @nabeel-servcorp 

See https://github.com/serilog-tracing/serilog-tracing?tab=readme-ov-file#enabling-tracing-with-activitylistenerconfigurationtracetosharedlogger

You can choose not to record traces for HTTP requests but then you won't be able to view traces. If you don't want to see these spans in your event list you can use a signal to exclude them.

![image](https://github.com/datalust/seq-tickets/assets/271514/7ff3068d-55a6-4ea0-bbf3-6286d124eec1)

**Reply by nblumhardt** [reply]
Using:

```csharp
    .Instrument.WithDefaultInstrumentation(false)
```

will turn off `HttpClient` instrumentation entirely, while leaving everything else intact. As Liam suggested, filtering these particular ones out is likely a better strategy.

The `StatusCode` property on the spans should explain why the requests are failing.

I just want to jump in and point out that the library you're using is maintained at https://github.com/serilog-tracing/serilog-tracing; it's better to raise SerilogTracing discussions/issues over there so that they're tracked alongside the relevant source code/releases etc. - thanks!

HTH!

**Reply by nabeel-servcorp** [reply]
Good advice about the issue posting on library page. I am taking this discussion over there now.
